Modules/@babylonjs/react-native/shared/BabylonNative.h

```diff
@@ -8,7 +8,7 @@ namespace Babylon
 
     void Initialize(facebook::jsi::Runtime& jsiRuntime, Dispatcher jsDispatcher);
     void Deinitialize();
-    void UpdateView(void* windowPtr, size_t width, size_t height, void* windowTypePtr = nullptr);
+    void UpdateView(void* windowPtr, size_t width, size_t height);
```

Comment:
Ideally, we should be removing `void *` from this header as well, in favor of WindowType.

Currently I cannot include the <Babylon> headers here without build failure, I'll keep tooling with CMake to so that this works, but it may require making this a library instead of a collection of source files included in another library. I'm no CMake expert, however.

Reply:
id suggest redefining the type here or using a void pointer for the time being. We can update include directories for the native modules to track down the headers required to access the window type definition. But right now the native modules aren't precompiled binaries when distributed through npm. So if BabylonNative.h references babylon native headers and our react-native native module references BabylonNative.h, i'm pretty sure wed have to package and distribute all headers required to resolve the windowtype, which is going to be a lot more work/seems beyond the scope of fixing breaking changes.

Reply:
After chatting offline, I'll go forward with retaining void*.